docs/pages/web/buttonsocial.tsx

```diff
@@ -0,0 +1,118 @@
+import { ButtonSocial } from 'gestalt';
+import AccessibilitySection from '../../docs-components/AccessibilitySection';
+import CombinationNew from '../../docs-components/CombinationNew';
+import docGen, { type DocGen } from '../../docs-components/docgen';
+import GeneratedPropTable from '../../docs-components/GeneratedPropTable';
+import MainSection from '../../docs-components/MainSection';
+import Page from '../../docs-components/Page';
+import PageHeader from '../../docs-components/PageHeader';
+import QualityChecklist from '../../docs-components/QualityChecklist';
+import SandpackExample from '../../docs-components/SandpackExample';
+import main from '../../examples/buttonsocial/main';
+
+export default function DocsPage({ generatedDocGen }: { generatedDocGen: DocGen }) {
+  return (
+    <Page title={generatedDocGen?.displayName}>
+      <PageHeader description={generatedDocGen?.description} name={generatedDocGen?.displayName}>
+        <SandpackExample code={main} hideEditor name="Main ButtonSocial example" />
+      </PageHeader>
+
+      <GeneratedPropTable generatedDocGen={generatedDocGen} />
+
+      <MainSection name="Usage guidelines">
```

Comment:
remove UNLESS design gave them to you. 

Keep just variant and localization. The Usage and best practices is designed-owned content